tests/lib/rules/no-missing-import.js

```diff
@@ -156,59 +156,133 @@ ruleTester.run("no-missing-import", rule, {
 
         // typescriptExtensionMap
         {
+            // name: "settings.node - [] as react - d.ts as d.js",
```

Comment:
👍 I like having the `name` field, it does not have to be in comments - per the [eslint docs](https://eslint.org/docs/latest/integrate/nodejs-api#ruletester)?

Reply:
It actually does because the eslint v7 tests throw an error 😢

I ended up commenting them here: [`b0de4ae` (#114)](https://github.com/eslint-community/eslint-plugin-n/pull/114/commits/b0de4ae6c477a303504a3220447c014c82697bd3)

Reply:
looks like it was introduced in an eslint v8 release. 😄